Docs/README.md

```diff
@@ -0,0 +1,10 @@
+# Documentation
+
+Here you can find documentation specific for using CreateAPI.
+
+- [Tutorial: Generating an API with CreateAPI](./Tutorial.md)
```

Comment:
Have you considered generating the documentation using DocC?

Reply:
It crossed my mind, but I haven't looked into it too much. Something totally worth doing in the future though once we've build up and stabilised the docs I think 👍